src/routes/ImprintScreen.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { ReactElement } from 'react'
+import styled from 'styled-components/native'
+
+import Link from '../components/Link'
+
+const Root = styled.View`
+  background-color: ${prop => prop.theme.colors.background};
+  height: 100%;
+  width: 100%;
+`
+
+const Container = styled.View`
+  margin: ${props => props.theme.spacings.sm};
+`
+
+const Description = styled.Text`
```

Comment:
If Andys PR (LUN-247) gets merged first, would be nice if you can use the text-components he created there.